generator/src/main/java/com/scottlogic/deg/generator/violations/ManifestWriter.java

```diff
@@ -1,7 +1,6 @@
-package com.scottlogic.deg.generator.outputs.manifest;
+package com.scottlogic.deg.generator.violations;
```

Comment:
Similar to the comment above, this file should remain in the `generator` module, but it makes sense to be in an `outputs` package as that is what it relates to.

Reply:
changed back to how it was